test/functional/api/cas/cache.py

```diff
@@ -182,6 +182,7 @@ def standby_detach(self, shortcut: bool = False):
         return casadm.standby_detach_cache(cache_id=self.cache_id, shortcut=shortcut)
 
     def standby_activate(self, device, shortcut: bool = False):
+        dev_path = device.path if device is Device else device
```

Comment:
In general, all method for test-framework API accept only one type as the valid one and I feel like changing this approach now may lead to confusion. I suggest accepting `device` either as string or `Device`.